src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,59 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_10 = 10;
+
+    /**
+     *
+     */
+    public FizzBuzz() {
+    }
+
+    /**
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        return basic(rango);
+    }
+
+    /**
+     * The Two Parts.
+     *
+     * @param rango int[].
+     * @return arrayObj2 Object[].
+     */
+    public Object[] fizzBuzz2(final int[] rango) {
+        Object[] arrayObj2 = basic(rango);
+        for (int i = 0; i < arrayObj2.length; i++) {
+            arrayObj2[i] = arrayObj2[i].equals("FizzBuzz") || arrayObj2[i].equals("Fizz")
+                    || arrayObj2[i].equals("Buzz") ? arrayObj2[i]
+                    : (Integer) arrayObj2[i] % NUMBER_10 == NUMBER_5 ? "Buzz"
+                    : ((Integer) arrayObj2[i] / NUMBER_10) % NUMBER_10 == NUMBER_5 ? "Buzz"
+                    : (Integer) arrayObj2[i] % NUMBER_10 == NUMBER_3 ? "Fizz"
+                    : ((Integer) arrayObj2[i] / NUMBER_10) % NUMBER_10 == NUMBER_3 ? "Fizz" : arrayObj2[i];
+        }
+        return arrayObj2;
+    }
+    /**
+     * Method for create array.
+     *
+     * @param rango int[].
+     * @return arrayObj Object[].
+     */
+    private Object[] basic(final int[] rango) {
+        Object[] arrayObj = new Object[rango[1] - rango[0] + 1];
+        int n = rango[0];
+        for (int i = 0; i < arrayObj.length; i++) {
+            arrayObj[i] = n % NUMBER_5 == 0 ? n % NUMBER_3 == 0 ? "FizzBuzz" : "Buzz"
+                    : n % NUMBER_3 == 0 ? "Fizz" : n;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)